airflow/cli/commands/standalone_command.py

```diff
@@ -206,7 +208,7 @@ def is_ready(self):
         Detects when all Airflow components are ready to serve.
         For now, it's simply time-based.
         """
-        return self.port_open(8080) and self.job_running(SchedulerJob) and self.job_running(TriggererJob)
+        return self.port_open(self.web_server_port) and self.job_running(SchedulerJob) and self.job_running(TriggererJob)
```

Comment:
This logic needs to be reviewed and fixed, port_open is called multiple times because of waiting for scheduler job. the socket can be opened after this condition is true - 
`self.job_running(SchedulerJob) and self.job_running(TriggererJob)`
to avoid opening and closing socket multiple times.